docs/source/notebook.rst

```diff
@@ -192,8 +200,9 @@ operations within the notebook, by clicking on an icon.
     As of notebook version 4.1, the user interface allows for multiple cells to
     be selected.  The ``quick celltype selector``, found in the menubar, will
     display a dash ``-`` when multiple cells are selected to indicate that the
-    type of the cells in the selection might not be unique. The quick selector
-    can still be used to change the type of the selection and will change the
+    type of the cells in the selection might not be unique.
+    In that case the quick selector
+    will change the type of the selection and will change the
```

Comment:
Suggested wording:

As of notebook version 4.1, the user interface allows for multiple cells to
be selected.  The ``quick celltype selector``, found in the menubar, will
display a dash ``-`` to indicate when multiple cells are selected. There is also highlighting of the cells.